test/repository_test.rb

```diff
@@ -115,6 +127,23 @@
           UserRepository.create(user1)
           user1.id.must_equal id
         end
+
+        describe 'when entity is already persisted' do
+          it 'assigns and persists created_at attribute' do
+            result = PhoneRepository.create(@phone_without_created_at)
+            time_now = result.created_at
+            result.created_at.must_equal time_now
+          end
+
+          it 'assigned and persisted created_at' do
+            result = PhoneRepository.create(@phone_with_created_at)
+            result.created_at.must_equal @created_at
```

Comment:
this assertion is wrong, please fix your fixture to make it a persisted entity by using Repository